docs/en/guides/pipeline-integrations/getting-started/installing_app.md

```diff
@@ -406,11 +406,11 @@ apps.tk-multi-pythonconsole.location:
   type: app_store
   name: tk-multi-pythonconsole
   version: v1.1.2
-```' %}
```

Comment:
Is this deletion on purpose?  I looks like it matches the {% on line 381?  It might have to go on its own line.

Reply:
This should be cleaned up now